package.json

```diff
@@ -6,6 +6,7 @@
   "private": false,
   "scripts": {
     "dist": "scripts/dist",
+    "build": "waffle",
```

Comment:
what's the difference between `build` and `dist`? We should document it in the readme

Reply:
went with just dist, since it is required in the git pipeline